distribution/lib/Standard/Base/0.0.0-dev/src/System/File.enso

```diff
@@ -473,13 +553,13 @@ type File
 
              example_delete =
                  file = Examples.data_dir / "my_file"
-                 file.write "hello"
+                 file.write_text "hello"
                  file.delete
     delete : Nothing ! File_Error
     delete =
-        here.handle_java_exceptions this <|
-            this.prim_file.delete
-        Nothing
+        result = here.handle_java_exceptions this this.prim_file.delete
+        if result.is_error then result else
+            Nothing
```

Comment:
```suggestion
        result = here.handle_java_exceptions this this.prim_file.delete
        if result.is_error then result else
            Nothing
```

I'm wondering if the check is needed -  IMO we can just return the value - `EnsoFile.delete` returns `void` which gets converted to a `Nothing` at Enso side.